.github/workflows/validation.yaml

```diff
@@ -1,23 +1,22 @@
 name: Lint, Build, Test
 
 on:
-  push:
   pull_request:
 
 jobs:
   lint-build-test:
     strategy:
       matrix:
-        node: ['14.x', '16.x', '18.x']
+        node: ['14.x', '16.x', '18.x', '20.x']
```

Comment:
tbh we could probably remove testing for 14.x and 16.x
we should probably only support the two latest LTS we have on our ecosystem, wdyt?